### PR TITLE
feat: gh action for hugo

### PR DESCRIPTION
**Pull request by k-burt-uch**
Link to JIRA ticket if there is one: 

### New Features
- Replace travis with GH Actions

### CODE REVIEW

File: .github/hugo_ci.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: Hugo Build and Deploy
+
+on:
+  push:
+    branches:
+      - master
```
Comment (paulineribeyre) [inline .github/hugo_ci.yaml:6]:
```suggestion
on: push
```

Comment (paulineribeyre) [reply .github/hugo_ci.yaml:6]: ^ to support the `alpha.gen3.org` flow and so we can test this PR

File: .github/hugo_ci.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    - name: Build Site
+      run: hugo
+
+    - name: Install AWS CLI
+      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+      uses: chrislennon/action-aws-cli@1.1
+
+    - name: Deploy to S3
+      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```
Comment (paulineribeyre) [inline .github/hugo_ci.yaml:40]: i think we can remove this `if` statement

File: .github/hugo_ci.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      uses: actions/checkout@v3
+
+    - name: Setup Hugo
+      uses: peaceiris/actions-hugo@v2
+      with:
+        hugo-version: '0.100.2'
+        extended: true
+
+    - name: Setup Asciidoctor
+      run: sudo apt-get install -y asciidoctor
```
Comment (paulineribeyre) [inline .github/hugo_ci.yaml:27]:
```suggestion
    - name: Setup Asciidoctor
      run: sudo apt-get install -y asciidoctor # support asciidoc files
```

**Review by wangfan860 — APPROVED** [review]

**Review by paulineribeyre — APPROVED** [review]
nice!!

**Comment by paulineribeyre** [comment]
i adjusted the repo setting to require this new check instead of the travis one

**Pull request merged**